feature/policy_forwarding/encapsulation/otg_tests/staticgueencap_and_bgp_path_selection/README.md

```diff
@@ -0,0 +1,382 @@
+       * PF-1.6.4: Migrate routing of AF2 flows from ATE1:Port1 to ATE2:Port1 to ATE2:Port3 instead
+           * So now BE1-AF2 are migrated
+       * PF-1.6.5: Migrate routing of AF3 flows from ATE1:Port1 to ATE2:Port1 to ATE2:Port3 instead
+           * So now BE1-AF3 are migrated
+       * PF-1.6.6: Migrate routing of AF4 flows from ATE1:Port1 to ATE2:Port1 to ATE2:Port3 instead
+           * So now BE1-AF4 are migrated.<br>
+
+   * Expectations:
+       * Same as above in PF-1.6.2 for each traffic class migrated.<br><br><br>
+
+**PF-1.6.7: Encap all traffic flows from ATE2 to ATE1:Port1**
+   * Situation:
+       * Test starts from the situation as in PF-1.6.3 above. Therefore all traffic from ATE1 to ATE2 is being encaped in the DUT and routed towards ATE2:Port3
+
+   * Test Steps:
+       * Please run all the health checks suggested above prior to starting the test as a baseline.
+       * Start all flows above:
+           * Dont start the above mentioned Unencaped flows from ATE2:Port1<br>
+    
+   * Expectations:
+       * Flows from ATE1 to ATE2 will be GUE encaped to tunnel destination IPv4Prefix12/28 and routed out to ATE2:Port3. The flows must be successful. Ensure that the TTL on the outer heder is decremented and lower than that of the inner header. TOS byte value in the outer header is the same as the TOS value in the inner header.
+       * Ensure the DUT is streaming accurate data on the number of packets/bytes being Encaped.
+       * ATE2:Port3 to ATE1:Port1 Encaped flows with tunnel destination as IPv4Prefix11/28 must be 100% successful. The flows are expected to route through the IBGP peering between ATE2:Port1 --> DUT:Port2 --> ATE
+       * Ensure the DUT is streaming accurate data on the number of packets/bytes being Decaped.
+       * Run post test Healthchecks and compare. Ensure there are no drops, coredumps etc.<br><br><br>
+
+
+**PF-1.6.8: [Negative scenario] EBGP route to the tunnel destination "IPv4Prefix12/28" is removed inflight**
+   * Situation:
+       * Test start from the end state of "PF-1.6.7" whereby the DUT is encaping all traffic from ATE1 to ATE2 to tunnel destination address "IPv4Prefix12/28". Also, all traffic in the direction from ATE2 to ATE1 is being encaped to tunnel destination address "IPv4Prefix11/2". ATE2 is not sending any unencaped flows.
+   * Test Steps:
+       * Please run all the health checks suggested above prior to starting the test as a baseline.
+       * Start all flows above:
+           * Dont start the above mentioned Unencaped flows from ATE2:Port1
+       * Configure the EBGP on ATE2:Port3 to stop advertising "IPv4Prefix12/28" to DUT:Port4. When this happens, following is expected.
+           * The Static route that points the VPNH to the tunnel destination (NHG or tunnel interface) must become invalid. Therefore, all the routes that are programmed by ATE2:Port2 with a VPNH of "IPv4Prefix17" in its IBGP session with DUT:Port3 must become invalid.
```

Comment:
It'd be useful in these test cases to define the configuration -- the static route here is specifically:

> static
>   dst: `IPv4Prefix17` 
>   nhop: `IPv4Prefix12`   # what address is actually the NH? this is a /28

so `IPv4Prefix12/28` is withdrawn, and `IPv4Prefix17` (which I assume is a /32) which is the VPNH becomes invalid, and hence at that point prefixes for whom the next-hop is set to `IPv4Prefix17` become invalid.

Given this scenario, you use the alternate routing.

It seems worth clarifying this because it's very easy to get lost in what the scenario here is IMHO.